libp2p/connmanager.nim

```diff
@@ -113,6 +113,9 @@ proc connectedPeers*(c: ConnManager, dir: Direction): seq[PeerId] =
       peers.add(peerId)
   return peers
 
+proc getConnections*(c: ConnManager): Table[PeerId, seq[Muxer]] =
```

Comment:
Should be easy to add a test around here:
https://github.com/status-im/nim-libp2p/blob/af5299f26c195540d3fbe0f1a20742658f8a6220/tests/testconnmngr.nim#L31-L43